src/localizations/en.json

```diff
@@ -117,6 +120,7 @@
     "scopes": "Scopes",
     "search_for_buisness": "Search by organisation number",
     "search_for_api": "Search for API",
+    "search_for_api_no_result": " {{count}} hits",
```

Comment:
Since the text key specifies this as no results - doesn't that mean that the `count` is redundant?
Or that the text key is misleading?

Reply:
Agree. I originally made the text for displaying number of search results, hence the {{count}}. I'll remove it!